dataflux_core/tests/fake_gcs.py

```diff
@@ -102,25 +110,12 @@ def download_as_bytes(self, retry=None):
 
     def open(self, mode: str, ignore_flush: bool = False):
         if mode == "rb":
-            return BlobReader()
+            return io.BytesIO(self.content)
```

Comment:
blob.open with "rb" should return a BlobReader class where further `.read()` on it returns the content. Is this desired?

Reply:
This is mainly to test what was saved to content with the FakeBlobWriter can then be read. It works this way with the Lightning Unit tests I have in the other repository.

Reply:
I'd suggest the test in the other repository to be modified so it takes a FakeBlobReader and perform "read" on it to get the content rather than bypassing it here. Reason is that with this, we are creating a major separation between how it's supposed to work vs what we wanted. For "r" mode we are returning a bytes stream and for "w" we are returning a writer object.

Reply:
II misunderstood what you were referring to when saying it returns the content. This fake implementation still returns a reader, a BytesIO reader, that torch.load can use to interact with the blob.content. 

I attempted implementing a FakeBlobReader but it became needlessly complicated for the role we want the fake to play in these tests.

Reply:
oh right. okay sounds good then. I had meant a FakeBlobReader to get closer to the actual invocation which will return a BlobReader class which we can fake it. Taking a step back I'd think we are okay here as they are fakes anyways.